packages/@dynamic-labs-connectors/intersend-evm/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@dynamic-labs-connectors/intersend-evm",
+  "version": "4.0.0-alpha.0",
+  "description": "Intersend EVM connector for Dynamic SDK",
+  "type": "module",
+  "main": "./src/index.js",
+  "typings": "./src/index.d.ts",
+  "publishConfig": {
+    "access": "public"
+  },
+  "private": false,
+  "exports": {
+    ".": {
+      "import": "./src/index.js",
+      "require": "./src/index.js",
+      "types": "./src/index.d.ts"
+    }
+  },
+  "peerDependencies": {
+    "@dynamic-labs/ethereum": "^4.0.0-alpha.24",
+    "@dynamic-labs/ethereum-core": "^4.0.0-alpha.24",
+    "@dynamic-labs/utils": "^4.0.0-alpha.24",
+    "@dynamic-labs/wallet-book": "^4.0.0-alpha.24",
+    "@dynamic-labs/wallet-connector-core": "^4.0.0-alpha.24",
```

Comment:
please change these to `^4.0.0-alpha.35` and remove `"@dynamic-labs/wallet-book"`